irohad/torii/processor/impl/query_processor_impl.cpp

```diff
@@ -28,23 +27,32 @@ namespace iroha {
         std::shared_ptr<validation::StatelessValidator> stateless_validator)
         : qpf_(std::move(qpf)), validator_(stateless_validator) {}
 
-    void QueryProcessorImpl::queryHandle(std::shared_ptr<model::Query> query) {
-      // if not valid send wrong response
-      if (!validator_->validate(*query)) {
-        model::ErrorResponse response;
-        response.query_hash = iroha::hash(*query);
-        response.reason = model::ErrorResponse::STATELESS_INVALID;
+    void QueryProcessorImpl::queryHandle(
+        std::shared_ptr<shared_model::interface::Query> qry) {
+      auto query = qry->makeOldModel();
```

Comment:
the same `shared_ptr<shared_model::proto::QueryResponse> query(qry->makeOldModel());`